src/components/AutoComplete/AutoComplete.stories.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { inlineTemplate } from '../../../storybook/decorators/storyTemplates';
+
+import AutoComplete from './AutoComplete';
+import {
+  MockAutoComplete,
+  basicExampleSource
+} from './__mocks__/mockAutoComplete';
+
+storiesOf('AutoComplete', module).add(
+  'AutoComplete',
+  withInfo({
+    source: false,
+    propTables: [AutoComplete],
+    propTablesExclude: [MockAutoComplete],
```

Comment:
@priley86 This approach can fix some problems we are facing with the storybook?

@danseethaler :+1: 

@gilad215 /cc

Reply:
👍  yes - that is the suggestion right now... nice job.